app/assets/stylesheets/general/_base.sass

```diff
@@ -103,3 +103,7 @@ a.underline
 
 .icon-space-l-half
   margin-left: 0.25em
+
+.api-section-top-margin
+  margin-top: 12%
+
```

Comment:
At the end of `app/assets/stylesheets/home.sass` should work, before the media queries start